nyaa/views/torrents.py

```diff
@@ -147,9 +148,73 @@ def edit_torrent(torrent_id):
 
         return flask.render_template('edit.html',
                                      form=form,
+                                     delete_form=delete_form,
                                      torrent=torrent)
 
 
+@bp.route('/view/<int:torrent_id>/delete', endpoint='delete', methods=['POST'])
+def edit_torrent(torrent_id):
```

Comment:
function name should be changed to something like `delete_torrent` or `soft_delete_torrent`